app/src/main/AndroidManifest.xml

```diff
@@ -5,6 +5,7 @@
     <application
         android:name=".main.app.PicoverApplication"
         android:allowBackup="true"
+        android:configChanges="uiMode"
```

Comment:
Why it is introduced ?

Reply:
I was verifying the solution proposed in: https://github.com/google/accompanist/issues/1493 for an issue with BottomSheet